common/src/main/scala/explore/Icons.scala

```diff
@@ -130,18 +130,6 @@ object Icons {
   @JSImport("@fortawesome/pro-solid-svg-icons", "faCheck")
   val faCheck: FAIcon = js.native
 
-  @js.native
-  @JSImport("@fortawesome/pro-solid-svg-icons", "faSort")
```

Comment:
Are these provided by prime?

Reply:
We are still using the same FA icons, but they are declared in `lucuma-ui` now.